non_www/tabs/school/download_national.html

```diff
@@ -9,33 +9,34 @@
       <div class="col-md-12 text-left" style="padding-top:10px;">
         <h3>National Data  - school layer</h3>
         <p>This page contains data downloads for the schools layer for the whole of England. These are available as CSV files containing baseline and scenario attribute data for zones and schools, and as geographic files that can be read by R (.Rds) or GIS programs such as QGIS (.geojson).</p>
-        <p>Please be aware that equivalent files for different purposes and geographies have <b>the same file name</b> when downloaded (e.g. ‘z_all.geojson’ for national zones files for both commuting and school travel).  To avoid confusion you may want to rename those files or save them in labelled folders. Note also that when <b>downloading 'csv' files using a Firefox browser</b>, the file will open onscreen in a new webpage.  If you then type "Ctrl + S" the onscreen data will download as a csv file.</p>
+        <p>Please be aware that equivalent files for different purposes and geographies have <b>the same file name</b> when downloaded (e.g. 'z_all.geojson' for national zones files for both commuting and school travel).  To avoid confusion you may want to rename those files or save them in labelled folders. Note also that when <b>downloading 'csv' files using a Firefox browser</b>, the file will open onscreen in a new webpage.  If you then type "Ctrl + S" the onscreen data will download as a csv file.</p>
       </div>
 
       <div class="col-md-12 text-left" style="padding-top:10px;">
         <h4><a name="natschoolzones"></a>School travel data for zones</h4>
-        <p>Download anonymised National School Census 2011 baseline data and scenario attribute data for all zones in England. These files present aggregate data for all children <b>living in a given zone</b>. </p>
+        <p>Download anonymised National School Census 2011 baseline data and scenario attribute data for all zones in England. These files present aggregate data for all children <b>living in a given zone</b></p>
+        <p>In the LSOA downloads, baseline and scenario data are not presented for cells containing 1 or 2 children.  Instead these data are set as missing ('NA').  These data are, however, included in the local authority downloads.</p>
```

Comment:
Nice, do we only have `NA` in these circumstances, i.e. do we have 0 if there are no children (or schools)?

Reply:
yes, have a look e.g. here: https://github.com/npct/pct-outputs-regional-notR/blob/master/school_public/lsoa/isle-of-wight/z_attributes.csv 
if there are no children then we do have 0, since this is allowed by the NPD SDC rules